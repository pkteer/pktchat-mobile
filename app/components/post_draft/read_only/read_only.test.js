// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import Preferences from '@mm-redux/constants/preferences';
import {shallowWithIntl} from '@test/intl-test-helper';

import ReadOnly from './index';

describe('PostDraft ReadOnly', () => {
    const baseProps = {
        testID: 'post_draft.read_only',
        theme: Preferences.THEMES.denim,
    };

    test('should match snapshot', () => {
        const wrapper = shallowWithIntl(<ReadOnly {...baseProps}/>);

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
