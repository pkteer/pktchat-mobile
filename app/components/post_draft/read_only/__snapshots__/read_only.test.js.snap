// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostDraft ReadOnly should match snapshot 1`] = `
<RNCSafeAreaView
  edges={
    Array [
      "bottom",
    ]
  }
  style={
    Object {
      "backgroundColor": "rgba(63,67,80,0.04)",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderTopColor": "rgba(63,67,80,0.2)",
        "borderTopWidth": 1,
        "flexDirection": "row",
        "height": 50,
        "paddingHorizontal": 12,
      }
    }
    testID="post_draft.read_only"
  >
    <CompassIcon
      color="#3f4350"
      name="glasses"
      style={
        Object {
          "fontSize": 20,
          "lineHeight": 22,
          "opacity": 0.56,
        }
      }
    />
    <InjectIntl(FormattedText)
      defaultMessage="This channel is read-only."
      id="mobile.create_post.read_only"
      style={
        Object {
          "color": "#3f4350",
          "fontSize": 15,
          "lineHeight": 20,
          "marginLeft": 9,
          "opacity": 0.56,
        }
      }
    />
  </View>
</RNCSafeAreaView>
`;
